pkg/generic/thrift/http.go

```diff
@@ -77,12 +108,83 @@ func (r *ReadHTTPResponse) SetBase64Binary(enable bool) {
 	r.base64Binary = enable
 }
 
+// SetUseRawBodyForHTTPResp ...
+func (r *ReadHTTPResponse) SetUseRawBodyForHTTPResp(useRawBodyForHTTPResp bool) {
+	r.useRawBodyForHTTPResp = useRawBodyForHTTPResp
+}
+
+// SetDynamicGo ...
+func (r *ReadHTTPResponse) SetDynamicGo(cv t2j.BinaryConv, msg remote.Message) {
+	r.cv = cv
+	r.msg = msg
+	r.dynamicgoEnabled = true
+}
+
 // Read ...
 func (r *ReadHTTPResponse) Read(ctx context.Context, method string, in thrift.TProtocol) (interface{}, error) {
```

Comment:
this function is toooo long, consider refactoring into smaller ones

Reply:
done